    To login, `huggingface_hub` requires a token generated from https://huggingface.co/settings/tokens .
Token can be pasted using 'Right-Click'.
Token:
Add token as git credential? (Y/n) y
Token is valid.
Traceback (most recent call last):
  File "C:\Program Files\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Program Files\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\ai art amd\amd_venv\Scripts\huggingface-cli.exe\__main__.py", line 7, in <module>
  File "C:\ai art amd\amd_venv\lib\site-packages\huggingface_hub\commands\huggingface_cli.py", line 47, in main
    service.run()
  File "C:\ai art amd\amd_venv\lib\site-packages\huggingface_hub\commands\user.py", line 117, in run
    login()
  File "C:\ai art amd\amd_venv\lib\site-packages\huggingface_hub\_login.py", line 92, in login
    interpreter_login()
  File "C:\ai art amd\amd_venv\lib\site-packages\huggingface_hub\_login.py", line 150, in interpreter_login
    _login(token=token, add_to_git_credential=add_to_git_credential)
  File "C:\ai art amd\amd_venv\lib\site-packages\huggingface_hub\_login.py", line 244, in _login
    if _is_git_credential_helper_configured():
  File "C:\ai art amd\amd_venv\lib\site-packages\huggingface_hub\_login.py", line 264, in _is_git_credential_helper_configured
    helpers = list_credential_helpers()
  File "C:\ai art amd\amd_venv\lib\site-packages\huggingface_hub\utils\_git_credential.py", line 37, in list_credential_helpers
    output = run_subprocess("git config --list", folder=folder).stdout
  File "C:\ai art amd\amd_venv\lib\site-packages\huggingface_hub\utils\_subprocess.py", line 61, in run_subprocess
    return subprocess.run(
  File "C:\Program Files\Python310\lib\subprocess.py", line 501, in run
    with Popen(*popenargs, **kwargs) as process:
  File "C:\Program Files\Python310\lib\subprocess.py", line 969, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Program Files\Python310\lib\subprocess.py", line 1438, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
FileNotFoundError: [WinError 2] The system cannot find the file specified
